cluster/ephemeral-provider-common.sh

```diff
@@ -17,6 +18,7 @@ function prepare_config() {
     cat >hack/config-provider-$KUBEVIRT_PROVIDER.sh <<EOF
 master_ip=$(_main_ip)
 docker_tag=devel
+docker_tag_alt=devel-alt
```

Comment:
Would it make sense to add here a symbol which will not be accepted in k8s names?

Reply:
oh, interesting idea. that ensures that I don't have any logic that somehow results in the tag being in the name field of any object.

Reply:
fixed, interestingly enough, this did actually cause failures. So good suggestion :+1: